openwisp_monitoring/device/base/models.py

```diff
@@ -273,7 +273,7 @@ def related_metrics(self):
     @receiver(threshold_crossed, dispatch_uid='threshold_crossed_receiver')
     def threshold_crossed(sender, metric, alert_settings, target, first_time, **kwargs):
         """
-        Changes the health status of a device when a alert_settings is crossed.
+        Changes the health status of a device when a metric threshold is crossed.
```

Comment:
the threshold is not set at metric level though, but at alert settings level. What about: "when a threshold defined in the alert settings related to the metric is crossed"? Or something along this line.